### PR TITLE
improved button's tooltip consistency

### PR DESCRIPTION
**Pull request by nileshprasad137**
Pull Request for Issue #14949 .

### Summary of Changes
Removed` "title"` inside span for removing redundant tooltip

### Documentation Changes Required 
No

### CODE REVIEW

File: administrator/modules/mod_logged/tmpl/default.php [hunk 1]
```diff
@@ -17,7 +17,7 @@
 			<div class="span8">
 				<?php if ($user->client_id == 0) : ?>
 					<a title="<?php echo JHtml::_('tooltipText', 'MOD_LOGGED_LOGOUT'); ?>" href="<?php echo $user->logoutLink; ?>" class="btn btn-danger btn-mini hasTooltip">
-						<span class="icon-remove icon-white" title="<?php echo JText::_('JLOGOUT'); ?>"></span>
+						<span class="icon-remove icon-white" ></span>
```
Comment (Quy) [inline administrator/modules/mod_logged/tmpl/default.php]: Remove space before `>`.

Comment (nileshprasad137) [inline administrator/modules/mod_logged/tmpl/default.php]: Oh! yes.

**Comment by brianteeman** [comment]
you still need to have aria-hidden=true on the icon span or a screen reader will read out that value of the icon font.

**Review by brianteeman — APPROVED** [review]

**Comment by nileshprasad137** [comment]
Can be merged?

**Comment by jeckodevelopment** [comment]
No tests yet

**Comment by Quy** [comment]
I propose closing this pr in favor of #15068 as there is an a11y issue on which to use `aria-label` , `title`, `element-invisible`.

**Comment by Quy** [comment]
Please close as #15068 https://github.com/joomla/joomla-cms/pull/15068/commits/e969192a65ca2f0afcb26b157f0de41a719b6465 is more complete.

**Pull request closed without merging**